### DISCUSSION TITLE
Can't apply #__next full height to production mode

### DISCUSSION THREAD

**Discussion by ytliuSVN** [vercel/next.js#24305] [Help] [answered]
I'm working on my login page where I want to apply full height to pages globally. I've done this with [Tailwind CSS](https://github.com/ytliuSVN/nextjs-firechat/blob/main/styles/tailwind.css) like this:

```
// styles/tailwind.css
@layer base {
  html,
  body,
  #__next {
    @apply h-screen;
  }
}
```

When I load `localhost:3000`, `<div id="__next">` was applied successfully with full height of the viewport.

![fh (copy)](https://user-images.githubusercontent.com/13102859/115542879-5141d480-a2d3-11eb-9ab6-fc04bd50f071.png)

However, `#__next` div was not be set to full height in **[production mode](http://nextjs-firechat.vercel.app/)**.

Please visit **[NextJS FireChat](https://github.com/ytliuSVN/nextjs-firechat)** to view my application.

That's the whole point of the question. Any suggestions are welcome.

**Comment by bobarros** [comment]
Since you said any suggestions.  I don't have any knowledge regarding Tailwind.
You're already using useEffect inside your Home function. You could add another:

```javascript
useEffect(() => {

  if (document) {
    document.getElementById("#__next").style.height = "100vh";
  }

}, []);
```

**Comment by akellbl4** [comment, marked as answer]
Hi @ytliuSVN
I use code like this:
```css
html,
body {
	@apply h-full;
}

#__next {
	@apply flex flex-col min-h-full;
}
```

UPD. I've checked your project and discovered that if remove `@layer base` wrapper everything fine in production mode. I'm not an expert in tailwind but it works :)

**Reply by ytliuSVN** [reply]
Hi @akellbl4 
Thank you for your great help. I end up to solve with this:

```
html,
body,
#__next {
  @apply flex flex-col h-screen;
}
```

**Reply by adamhp** [reply]
Solved for me almost a year later too ;)

**Comment by MuhammadMinhaj** [comment]
Also, it can be solved with position absolute.
